tests/slab2_initial_composition_3d_shell.prm

```diff
@@ -0,0 +1,110 @@
+##### simple test for slab2 model as initial composition
+
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+
+set Adiabatic surface temperature          = 1613.0
+
+subsection Geometry model
+  set Model name = spherical shell
+
+  subsection Spherical shell
+    set Inner radius = 3481000
+    set Outer radius = 6336000
+  end
+end
+
+
+subsection Initial temperature model
+  set Model name = harmonic perturbation
+  subsection Harmonic perturbation
+    set Magnitude = 30
+    set Reference temperature = 1613
+  end
+end
+
+
+subsection Boundary temperature model
+  set List of model names = spherical constant
+  subsection Spherical constant
+    set Inner temperature = 3000
+    set Outer temperature = 273
+  end
+end
+
+
+# The parameters below this comment were created by the update script
+# as replacement for the old 'Model settings' subsection. They can be
+# safely merged with any existing subsections with the same name.
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = inner, outer
+end
+
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = inner:function, outer:function
+end
+
+subsection Boundary velocity model
+  subsection Function
+    set Function expression = 1;0;0
+  end
+end
+
+
+subsection Compositional fields
+  set Number of fields = 1
+  set Names of fields  = slabs
+  set Compositional field methods = static
+end
+
+
+subsection Initial composition model
+  set Model name = slab model
+  # The following lines needs to be modified such that
+  # they point to the location of the output slab2 model
+  # datafile, generated using the jupyter notebook in
+  # (contrib/python) folder in the aspect base directory.
+  subsection Slab model
+    set Data directory       = /home/postdoc/Documents/
+    set Data file name       = slab2_depth_thickness.txt
```

Comment:
We will have to find a way to make this test run on our tester machine. Can you include a very low resolution version of the slab files in `data/initial_composition/slab`? If that does not work, you could also make up a file with a new dataset (e.g. a single slab with a very large thickness). The important part is just that it can be included in the repository and can be used in this test.